src/runtime/opengl.cpp

```diff
@@ -464,7 +476,8 @@ WEAK KernelInfo *create_kernel(void *user_context, const char *src, int size) {
 
         const char *args;
         if ((args = match_prefix(line, kernel_marker))) {
-            // ignore
+            // set name
+            kernel->name = strstrip(strndup(args, next_line - args), next_line - args);
```

Comment:
This string looks like it's never freed. Is there somewhere modules are cleaned up? If it's intentionally leaked, please add a comment.